templates/Default/admin/Default/1.6/universe_create_locations.php

```diff
@@ -1,11 +1,16 @@
-Working on Galaxy : <?php echo $Galaxy->getName(); ?> (<?php echo $Galaxy->getGalaxyID() ?>)<br />
+Working on Galaxy : <?php echo $Galaxy->getName(); ?> (<?php echo $Galaxy->getGalaxyID() ?>)<br /><br />
+
+<?php
+foreach ($LocTypes as $category => $LocIDs) { ?>
+		<a href="javascript:;" onclick="window.location.hash='<?php echo $category; ?>'">Jump to <?php echo $category; ?></a><br /><?php
```

Comment:
Can you change this to
```html
<a href="'<?php echo $category; ?>">Jump to <?php echo $category; ?></a>
```
please, so that the jumping can be done natively, without needing javascript

Reply:
Unfortunately, linking to anchors using html will only work once. After you click one of these links, you will not be able to click another (see, e.g., https://stackoverflow.com/a/3163635). The javascript does not have this restriction.

If this were a part of the main game, I'd be with you 100% on insisting that there be a non-javascript alternative. But since this is just a convenience item for an admin tool, it seems harmless to me. But if you know how to implement the html workarounds to the one-click limitation, please go for it!

Reply:
@hemberger that link doesn't seem to suggest any problem with anchor # links, and I've never been aware of any issues (it certainly works fine in chrome on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a )

Reply:
I was referring to:
![image](https://user-images.githubusercontent.com/846186/31211148-b3f69b22-a94b-11e7-93cc-ca2038bde2ef.png)
If I use the code you suggested, the link can be clicked once, but after that anchor links will do nothing when clicked. I tested it to confirm, and I'm fairly certain I set it up correctly -- I can set up again on the test server to show you if you'd like.